### PR TITLE
SNMP Capabilities

### PR DESCRIPTION
**Pull request by murrant**
Allow all available auth algorithms to be set in global settings and elsewhere
cleanup other usages

Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by Jellyfrog** [comment]
Why have it in 1 place when you can have it in 10?

File: LibreNMS/SNMPCapabilities.php [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+    public static function cryptoAlgoritms(): array
+    {
+        $aes256 = self::supportsAES256();
+
+        return [
+            'AES' => true,
+            'AES-192' => $aes256,
+            'AES-256' => $aes256,
+            'AES-256-C' => $aes256,
```
Comment (Jellyfrog) [inline LibreNMS/SNMPCapabilities.php:92]: is this supported but not shown?
`  -x PROTOCOL		set privacy protocol (DES|AES|AES-192|AES-256)`

**Review by Jellyfrog — APPROVED** [review]

File: LibreNMS/SNMPCapabilities.php (hunk 1 above)
Comment (murrant) [reply LibreNMS/SNMPCapabilities.php:92]: Unclear

Comment (murrant) [reply LibreNMS/SNMPCapabilities.php:92]: https://community.librenms.org/t/cisco-snmpv3-using-aes-256/15168/11

**Pull request merged**